src/typescript/sdk/src/const.ts

```diff
@@ -59,7 +59,7 @@ export const QUOTE_VIRTUAL_CEILING = 1_400_000_000_000n;
 export const POOL_FEE_RATE_BPS = 25;
 export const MARKET_REGISTRATION_FEE = ONE_APTN;
 export const MARKET_REGISTRATION_DEPOSIT = 4n * ONE_APTN;
-export const MARKET_REGISTRATION_GAS_ESTIMATION_NOT_FIRST = 6000;
+export const MARKET_REGISTRATION_GAS_ESTIMATION_NOT_FIRST = ONE_APT * 0.005;
```

Comment:
Does it really cost this much? I think you can use 300,000 as a safe estimate (0.003). 200,000 is the default for most things now from what I remember, so ideally we'd just use that unless you found that it results in it failing (I think you mentioned this, not sure)

Reply:
![image](https://github.com/user-attachments/assets/bca1de92-54fa-4fcf-8a6a-890a7c236994)


I put this just to make sure.

Maybe we can remove after we do not use the old contract anymore. But for now, while I'm still working with the old one for these PRs, I need this.

Reply:
Gotcha, does the new one cost less gas? Thanks for making sure this works, appreciate the attention to detail here

Reply:
I think the new one uses less yes